frontend/www/js/omegaup/components/group/Form.vue

```diff
@@ -0,0 +1,119 @@
+        <div class="row">
+          <div class="form-group col-md-6">
+            <label class="control-label w-100">
+              {{ T.wordsName }}
+              <input
+                v-model="name"
+                name="title"
+                required
+                type="text"
+                class="form-control"
+                @blur="onGenerateAlias"
+              />
+            </label>
+          </div>
+
+          <div class="form-group col-md-6">
+            <label class="control-label w-100">
+              {{ T.contestNewFormShortTitleAlias }}
+              <input
+                v-model="alias"
+                name="alias"
+                required
+                type="text"
+                class="form-control"
+                disabled="true"
+              />
+            </label>
+            <p class="help-block">{{ T.contestNewFormShortTitleAliasDesc }}</p>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="form-group col-md-6">
+            <label class="control-label w-100">
+              {{ T.groupNewFormDescription }}
+              <textarea
+                v-model="description"
+                required
+                name="description"
+                cols="30"
+                rows="5"
+                class="form-control"
+              ></textarea>
+            </label>
+          </div>
+        </div>
+
+        <div class="form-group">
+          <button v-if="isUpdate" type="submit" class="btn btn-primary">
+            {{ T.groupNewFormUpdateGroup }}
+          </button>
+          <button v-else type="submit" class="btn btn-primary">
+            {{ T.groupNewFormCreateGroup }}
+          </button>
+        </div>
+      </form>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
+import T from '../../lang';
+import latinize from 'latinize';
+
+@Component
+export default class GroupForm extends Vue {
+  @Prop({ default: false }) isUpdate!: boolean;
+
+  T = T;
+  alias: string = '';
+  description: string = '';
+  name: string = '';
+
+  onSubmit(): void {
+    if (this.isUpdate) {
+      this.$emit('update-group', this.name, this.alias, this.description);
+      return;
+    }
+    this.$emit('create-group', this.name, this.alias, this.description);
+  }
+
+  onGenerateAlias(): void {
+    if (this.isUpdate) {
+      return;
+    }
+
+    // Remove accents
+    let generatedAlias = latinize(this.name);
+
+    // Replace whitespace
+    generatedAlias = generatedAlias.replace(/\s+/g, '-');
+
+    // Remove invalid characters
+    generatedAlias = generatedAlias.replace(/[^a-zA-Z0-9_-]/g, '');
+
+    generatedAlias = generatedAlias.substring(0, 32);
+
+    this.alias = generatedAlias;
+  }
+
+  @Watch('alias')
+  onValueChanged(newValue: string): void {
+    if (this.isUpdate) {
+      return;
+    }
+    this.$emit('alias-changed', newValue);
```

Comment:
```suggestion
    this.$emit('update:alias', newValue);
```

así es compatible con `:alias.sync="alias"`.

Reply:
Ah, más bien aquí el nombre está mal. 

Lo que se debería de emitir la validación de alias no utilizado. Así que con cambiar el nombre creo que está bien.